robot_calibration/src/plane_finder.cpp

```diff
@@ -143,13 +144,14 @@ bool PlaneFinder::find(robot_calibration_msgs::CalibrationData * msg)
     if (!std::isfinite(p.point.x) || !std::isfinite(p.point.y) || !std::isfinite(p.point.z))
       continue;
 
-    // Remove the points immediately in front of te camera in the point cloud
+    // Remove the points immediately in front of the camera in the point cloud
     // NOTE : This is to handle sensors that publish zeros instead of NaNs in the point cloud
     if (p.point.z == 0)
       continue;
 
     // Transformed point in the transformed frame (default is base_link))
-    if (p_out.point.z < min_z_ or p_out.point.z > max_z_ or p_out.point.y < min_y or p_out.point.y > max_y)
+    if (p_out.point.y < min_y or p_out.point.y > max_y or p_out.point.y < min_y or p_out.point.y > max_y or
```

Comment:
min_y_ -- you're missing underscore

Reply:
wait -- "or" ? That's not even C++?

Reply:
> wait -- "or" ? That's not even C++?

It actually is :) See [cppreference/Logical operators](http://en.cppreference.com/w/cpp/language/operator_logical).

Reply:
@gavanderhoorn thanks. Wow. In something like 15 years of C/C++ work, I don't think I've ever used that, or even knew it existed!

That said -- style guide says don't do it: https://google.github.io/styleguide/cppguide.html#Boolean_Expressions

And one other note @narora1  -- this has two comparisons for Y -- I think one of those was meant to be for X.